presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -27,12 +28,18 @@ public TestShowStats()
         super(() -> createQueryRunner(ImmutableList.of()));
     }
 
-    @Test
-    public void testShowStats()
+    @BeforeClass
+    public void setUp()
     {
         assertUpdate("CREATE TABLE nation_partitioned(nationkey BIGINT, name VARCHAR, comment VARCHAR, regionkey BIGINT) WITH (partitioned_by = ARRAY['regionkey'])");
         assertUpdate("INSERT INTO nation_partitioned SELECT nationkey, name, comment, regionkey from tpch.tiny.nation", 25);
+        assertUpdate("CREATE TABLE orders_partitioned(orderkey BIGINT, totalprice BIGINT, orderstatus BIGINT) WITH (partitioned_by = ARRAY['orderstatus'])");
```

Comment:
The schema of `orders` and `lineitem` tables in TPC-H is different.

```
presto:tiny> desc orders;
desc orders;
    Column     |    Type     | Extra | Comment 
---------------+-------------+-------+---------
 orderkey      | bigint      |       |         
 custkey       | bigint      |       |         
 orderstatus   | varchar(1)  |       |         
 totalprice    | double      |       |         
 orderdate     | date        |       |         
 orderpriority | varchar(15) |       |         
 clerk         | varchar(15) |       |         
 shippriority  | integer     |       |         
 comment       | varchar(79) |       |         
(9 rows)

Query 20181017_181406_00002_cjp6j, FINISHED, 4 nodes
Splits: 22 total, 22 done (100.00%)
0:00 [9 rows, 595B] [25 rows/s, 1.61KB/s]

presto:tiny> desc lineitem;
desc lineitem;
    Column     |    Type     | Extra | Comment 
---------------+-------------+-------+---------
 orderkey      | bigint      |       |         
 partkey       | bigint      |       |         
 suppkey       | bigint      |       |         
 linenumber    | integer     |       |         
 quantity      | double      |       |         
 extendedprice | double      |       |         
 discount      | double      |       |         
 tax           | double      |       |         
 returnflag    | varchar(1)  |       |         
 linestatus    | varchar(1)  |       |         
 shipdate      | date        |       |         
 commitdate    | date        |       |         
 receiptdate   | date        |       |         
 shipinstruct  | varchar(25) |       |         
 shipmode      | varchar(10) |       |         
 comment       | varchar(44) |       |         
(16 rows)

Query 20181017_181412_00003_cjp6j, FINISHED, 4 nodes
Splits: 22 total, 22 done (100.00%)
0:00 [16 rows, 1.04KB] [61 rows/s, 4KB/s]

presto:tiny> desc region;
desc region;
  Column   |     Type     | Extra | Comment 
-----------+--------------+-------+---------
 regionkey | bigint       |       |         
 name      | varchar(25)  |       |         
 comment   | varchar(152) |       |         
(3 rows)

Query 20181017_181425_00004_cjp6j, FINISHED, 4 nodes
Splits: 22 total, 22 done (100.00%)
0:00 [3 rows, 196B] [13 rows/s, 896B/s]

presto:tiny> desc nation;
desc nation;
  Column   |     Type     | Extra | Comment 
-----------+--------------+-------+---------
 nationkey | bigint       |       |         
 name      | varchar(25)  |       |         
 regionkey | bigint       |       |         
 comment   | varchar(152) |       |         
(4 rows)

Query 20181017_181429_00005_cjp6j, FINISHED, 4 nodes
Splits: 22 total, 22 done (100.00%)
0:00 [4 rows, 260B] [20 rows/s, 1.27KB/s]
```

Reply:
Thanks, which namespace has tiny schema? (how to activate presto:tiny> )